Launcher/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+using System;
+using System.Threading;
+using QuantConnect;
+using QuantConnect.Lean.Engine;
+
+namespace Launcher
+{
+    class Program
+    {
+        /// <summary>
+        /// Primary Analysis Thread:
+        /// </summary>
+        public static void Main(string[] args)
+        {
+            QuantConnect.Lean.Engine.Engine.Launch();
```

Comment:
The Engine should take config and run as an instance - I'd recommend moving the entire of the Main() method into the Launcher Main(). The call to engine should look something like this:

```
//Get the configuration
var config = ...;
//Launch the engine
var engine = new Engine(config);
engine.Run();
```